keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/CloudDriverCache.kt

```diff
@@ -38,4 +38,3 @@ interface CloudDriverCache {
 }
 
 class ResourceNotFound(message: String) : IntegrationException(message)
-class CacheLoadingException(message: String, cause: Throwable) : IntegrationException(message, cause)
```

Comment:
Moved this to keel-core so it can be used from anywhere that uses `CacheFactory`